### PR TITLE
updated supported models list table in readme

### PR DESCRIPTION
**Pull request by tthakkal**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,42 @@ limitations under the License.
...
+  - [Environment Variables](#environment-variables)
   - [Profiler](#profiler)
 
+
+## Tested Models and Configurations
+
+The following table contains models and configurations we have validated on Gaudi2.
+
+
+|  Model                 |  BF16        |             |  FP8         |             |
```
Comment (mandy-li) [inline README.md:38]: Can you merge the cell for `BF16` and the empty cell after it into one? Same for `FP8` cell

**Review by mandy-li — COMMENTED** [review]
Can you modify the table as I comment?

File: README.md (hunk 1 above)
Comment (tthakkal) [reply README.md:38]: Looks like its not possible in markdown

**Review by mandy-li — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -497,6 +496,20 @@ docker run -p 8080:80 \
...
 
+## TGI-Gaudi Benchmark
+
+### Static Batching Benchmark
+ To run static batching benchmark, please refer to [TGI's benchmark tool](https://github.com/huggingface/text-generation-inference/tree/main/benchmark).
+
+   To run it on the same machine, you can do the following:
+   * `docker exec -it <docker name> bash` , pick the docker started from step 2 using docker ps
+   * `text-generation-benchmark -t <model-id>` , pass the model-id from docker run command
+   * after the completion of tests, hit ctrl+c to see the performance data summary.
```
Comment (schoi-habana) [inline README.md:507]: worth mentioning the benchmark configs like "this benchmark runs the model with bs=[1, 2, 4, 8, 16, 32],  prompt len=10, decode len=8 by default. if you want to run other configs, please check text-generation-benchmark -h and run with those params"

many customers didn't notice the benchmark configuration is different from the server configuration.

Comment (tthakkal) [reply README.md:507]: added note

**Review by schoi-habana — APPROVED** [review]

**Pull request merged**